.github/workflows/main.yml

```diff
@@ -146,53 +192,22 @@ jobs:
           source /opt/rh/rh-python38/enable
           bazel test ${{ env.build_argv }} --test_summary=detailed --test_output=errors ${{ env.target }}
 
-
   release:
-    runs-on: ubuntu-latest
```

Comment:
we should build & release on centos7 & gcc 8. `hybridsql-base` should suffice.
Other linux env exist just for build test, which can be removed if not useful

Reply:
Yep. So I add `linux-build` job to build and test and pack zetasql  libraries when tag.
Other linux env exist just for build test, which can be removed if not useful